staging/src/github.com/labring/image-cri-shim/pkg/server/cri_server.go

```diff
@@ -60,7 +60,13 @@ func (s *server) ImageStatus(ctx context.Context,
 
 func (s *server) PullImage(ctx context.Context,
 	req *api.PullImageRequest) (*api.PullImageResponse, error) {
-	if req.Auth == nil {
+	logger.Debug("PullImage begin: %+v", req)
+	defaultImageDomain, _ := splitDockerDomain(req.Image.Image)
+	if req.Image != nil {
+		req.Image.Image = s.replaceImage(req.Image.Image, "PullImage")
+	}
+	afterReplaceImageDomain, _ := splitDockerDomain(req.Image.Image)
```

Comment:
`afterReplaceImageDomain` -> `replacedImageDomain`?

Reply:
> `afterReplaceImageDomain` -> `replacedImageDomain`?

I will fix it